dix/req.c

```diff
@@ -1402,6 +1408,20 @@ static void dix_rop(struct m0_dix_req *req)
 	M0_LEAVE();
 }
 
+/** Checks if the given cas get reply has a newer version of the value */
+static bool dix_item_version_cmp(struct m0_dix_item *ditem,
+				 struct m0_cas_get_reply *get_rep) {
```

Comment:
Parameters should be "const" presumably?

Reply:
Fixed